src/order-history/service.js

```diff
@@ -1,36 +1,25 @@
-/* eslint-disable no-console */
-// NOTE: console logs are intentionally added for REV-2577
-
 import { getAuthenticatedHttpClient, getAuthenticatedUser } from '@edx/frontend-platform/auth';
 import { getConfig } from '@edx/frontend-platform';
 
-const ECOMMERCE_RECEIPT_BASE_URL = `${process.env.RECEIPT_URL}`;
-
-const decimalishMatcher = /^([0-9]*[.,]*)+[0-9]*$/;
+const { ORDER_HISTORY_URL, RECEIPT_URL } = getConfig();
 
-function isNotDecimalish(num) {
-  if (Number.isNaN(num)) {
-    return true;
-  }
-
-  return !decimalishMatcher.test(num);
-}
+const ECOMMERCE_API_BASE_URL = `${ORDER_HISTORY_URL}`;
+const ECOMMERCE_RECEIPT_BASE_URL = `${RECEIPT_URL}`;
 
 // eslint-disable-next-line import/prefer-default-export
 export async function getOrders(page = 1, pageSize = 20) {
   const httpClient = getAuthenticatedHttpClient();
   const { username } = getAuthenticatedUser();
-  const { ORDER_HISTORY_URL } = getConfig();
 
-  const { data } = await httpClient.get(`${ORDER_HISTORY_URL}`, {
+  const { data } = await httpClient.get(`${ECOMMERCE_API_BASE_URL}`, {
```

Comment:
Continuation of the above suggestion:
```suggestion
  const { data } = await httpClient.get(ORDER_HISTORY_URL ? `${ECOMMERCE_API_BASE_URL}` : `${ECOMMERCE_API_BASE_URL}/orders/`, {
```